udata/core/followers/api.py

```diff
@@ -55,7 +55,7 @@ def get(self, id):
     @api.doc('follow', description=NOTE)
     def post(self, id):
         '''Follow an object given its ID'''
-        model = self.model.objects.only('id').get_or_404(id=id)
+        model = self.model.objects.get_or_404(id=id)
```

Comment:
The short term solution here is to add the slug in the requested fields: `only('id','slug')`.
The long term is to fix the SlugField to prevent its crash when field is not fetched.